spring-integration-core/src/main/java/org/springframework/integration/util/MessagingMethodInvokerHelper.java

```diff
@@ -391,7 +391,7 @@ else if (!Modifier.isPublic(method1.getModifiers())) {
 				return;
 			}
 			if (methodName == null
-					&& ObjectUtils.containsElement(new String[] { "start", "stop", "isRunning" }, method1.getName())) {
+					&& ObjectUtils.containsElement(new String[] {"start", "stop", "isRunning"}, method1.getName())) {
```

Comment:
Hmmm - my STS config adds these spaces - I guess it's not a checkstyle rule though.